launch_client.cmd

```diff
@@ -0,0 +1 @@
+.\build\windows\bin\profile\MultiplayerSample.GameLauncher.exe --console-command-file=launch_client.cfg
```

Comment:
Do we need to update VS debug settings as well?

Reply:
good catch, fixed!